### DISCUSSION TITLE
Old post was closed for reason that wasn't anything related to my question

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135719] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I asked if I could have multiple accounts on github due to my old account getting banned (for no reason). 
Does this apply to me?
```
One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine).
```
@LiteBrite82 responded with "The community is unable to assist with nor escalate your ticket..." and then preceded to close my discussion. I did not ask for help to assist in escalating my ticket. Please read the discussion before closing.

You can find the discussion here: https://github.com/orgs/community/discussions/134903#discussion-7019765

**Comment by Rodelph** [comment]
It is possible to have multiple accounts tho you will have to pay for them. Having org accounts would be much easier to set up and to manage.
